middleware/auth.go

```diff
@@ -150,9 +151,8 @@ func parseToken(r *http.Request) string {
 	return ""
 }
 
-func getOrCreateUser(exec boil.Executor, claims *IDTokenClaims) (*models.User, error) {
-	//TODO: wrap in transaction
-	user, err := models.Users(models.UserWhere.AccountsID.EQ(claims.Sub)).One(exec)
+func getOrCreateUser(db *sql.DB, claims *IDTokenClaims) (*models.User, error) {
+	user, err := models.Users(models.UserWhere.AccountsID.EQ(claims.Sub)).One(db)
```

Comment:
this should be inside the transaction

Reply:
fix
why becouse of atomic read and edit? 
but it's on the single tread all time

Reply:
hm...
I dont find othe solution
Maybe only add special client call that will create user